src/main/java/org/sciborgs1155/robot/Robot.java

```diff
@@ -27,6 +29,7 @@ public class Robot extends CommandRobot implements Logged, Fallible {
   private final CommandXboxController driver = new CommandXboxController(OI.DRIVER);
 
   // SUBSYSTEMS
+  @LogBoth private final Shooter shooter = Shooter.create();
```

Comment:
should be @LogFile (if you're logging something that implements Logged, use @LogFile. If you're logging a double, int, bool, etc., or something that implements Sendable, the you use @LogBoth)